### PR TITLE
Should be user

### PR DESCRIPTION
**Pull request by cskiwi**
_Motivation:_  As of the documentation this value should be a user. 
https://global.download.synology.com/download/Document/Software/DeveloperGuide/Firmware/DSM/All/enu/Synology_DiskStation_Administration_CLI_Guide.pdf (page 8)

`synoshare {--add} sharename share_desc share_path user_list_na user_list_rw user_list_ro share_browsable
adv_privilege`

*note*: This could be `$EFF_USER`. Not sure.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-_Package Version:_ 
+_Package Name:_
+_Package Version:_
 
-_NAS Model:_ 
-_NAS Architecture:_ 
-_DSM version:_ 
+_NAS Model:_
+_NAS Architecture:_
+_DSM version:_
```
Comment (hgy59) [inline .github/ISSUE_TEMPLATE.md]: please do not remove trailing spaces in this file.
If issue form is filled without space after _ the italic font of the subjects does not work.

**Comment by hgy59** [comment]
@cskiwi thanks for contributing to this repository.
For DSM7 related contributions you must checkout and use the dsm7 branch in your repository. 
We do not accept PRs for dsm 7 on master branch by now.
All about the current state of dsm7 support is discussed in #4215.

If the user permission fix is related to dsm 6 and older too, please create a dedicated PR for this.

**Comment by publicarray** [comment]
Ah part of it is my mistake, I rebased my dsm7 branch to get the latest changes from master, but I don't know how these commits have been added here?

**Comment by publicarray** [comment]
> If the user permission fix is related to dsm 6 and older too, please create a dedicated PR for this.

Yea that is what I meant in https://github.com/publicarray/spksrc/pull/25#issuecomment-753424596

**Comment by cskiwi** [comment]
Something went wrong with my master branch that's why it included the DSM7 stuff.
I'm trying again in: https://github.com/SynoCommunity/spksrc/pull/4357

**Pull request closed without merging**